### ISSUE TITLE
Copy of silverbucket/webfinger.js

### ISSUE THREAD

**Issue by silverbucket** [d-koppenhagen/webfinger#9]
Hi @d-koppenhagen any interest in merging your changes back upstream to https://github.com/silverbucket/webfinger.js ?

**Comment by d-koppenhagen** [comment]
Absolutely! I haven't had this side project on my list :D ... I froked it and embedded it in an angular app long time ago. So I think parts can be copied back but without the whole angular setup wich isn't at all necessary.

**Comment by silverbucket** [comment]
Hi @d-koppenhagen I have since migrated webfinger.js to typescript and made a number of improvements, as this repo was a direct copy of the original repo, yet it seems to be coming up on top in search results. Could you please update your readme to point back to the original repo? Or somehow indicate that this is a fork of the original repo?

**Comment by d-koppenhagen** [comment]
Sure. I added a note to the README and I will also archive this project since it is no longer maintained here.
